docs/add_threePids.md

```diff
@@ -1,5 +1,34 @@
 # Adding and removing ThreePids to an account
 
+<!--- TOC -->
```

Comment:
`<!--- TOC -->` and `<!--- END -->` have been added manually, all that is between are generated, or re-generated when calling `./gradlew knit`.